include/hpp/constraints/iterative-solver.hh

```diff
@@ -79,6 +83,60 @@ namespace hpp {
       };
     }
 
+    /// Solve a system of non-linear equations on a robot configuration
+    ///
+    /// The non-linear system of equations is built by adding equations with
+    /// method HierarchicalIterativeSolver::add.
+    ///
+    /// Note that a hierarchy between the equations can be
+    /// provided. In this case, the solver will try to solve the
+    /// highest priority equations first, and then to solve the lower priority
+    /// equations.
```

Comment:
I would mention (here or at HierarchicalIterativeSolver::add):

The priorities are by decreasing order (0 has the priority over 1).